src/model.js

```diff
@@ -396,10 +396,12 @@ module.exports = function(koop) {
         let xmax = tile2long(x+1, z);
         let ymin = tile2lat(y+1, z);
         let ymax = tile2lat(y, z);
-        xmin = xmin-Math.abs(xmin*buffer);
-        ymin = ymin-Math.abs(ymin*buffer);
-        xmax = xmax+Math.abs(xmax*buffer);
-        ymax = ymax+Math.abs(ymax*buffer);
+        let ydiff = ymax-ymin;
+        let xdiff = xmax-xmin;
+        xmin = xmin-Math.abs(xdiff*buffer);
+        ymin = ymin-Math.abs(ydiff*buffer);
+        xmax = xmax+Math.abs(xdiff*buffer);
+        ymax = ymax+Math.abs(ydiff*buffer);
         return {xmin:xmin, ymin:ymin, xmax:xmax, ymax:ymax};
```

Comment:
for simplicity, you can just return:

`return { xmin, ymin, xmax, ymax };`

Reply:
done!